t/Git.t

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/perl
 use lib::relative 'lib';
 use TestFor::Code::TidyAll::Git;
+local $ENV{LC_ALL} = 'C';
```

Comment:
There's no need for `local` here. Since this is file-scoped it doesn't go out of scope until the program exits anyway.